src/basic/unit-file.c

```diff
@@ -71,7 +71,7 @@ int unit_symlink_name_compatible(const char *symlink, const char *target, bool i
 }
 
 int unit_validate_alias_symlink_or_warn(int log_level, const char *filename, const char *target) {
-        const char *src, *dst;
+        _cleanup_free_ const char *src = NULL, *dst = NULL;
```

Comment:
remove the const here, it makes no sense as it is changed

Reply:
> remove the const here, it makes no sense as it is changed

In my local dialect, we say "follow who know road" meaning "let the experts guide you".
Thank you so much sir.

I want to quickly make some fixes recommended by @poettering in my other PR.
I'll push this soon.
Thank you sir.

Reply:
> remove the const here, it makes no sense as it is changed

Done sir.